src/test/java/com/stripe/functional/PlanTest.java

```diff
@@ -53,6 +57,21 @@ public void testRetrieve() throws StripeException {
     );
   }
 
+  @Test
+  public void testRetrieveIdWithForwardSlash() throws StripeException {
+    InvalidRequestException exception =
+        assertThrows(InvalidRequestException.class, () -> {
+          Plan.retrieve("Pro plan $699/month");
+        });
+    assertThat(exception.getMessage(), CoreMatchers.containsString(
+        "Unrecognized request URL (GET: /v1/plans/Pro+plan+$699/month)"));
+
+    verifyRequest(
+        ApiResource.RequestMethod.GET,
+        String.format("/v1/plans/%s", "Pro+plan+%24699%2Fmonth")
```

Comment:
Although the url-encoded is passed as `Pro+plan+%24699%2Fmonth` 
Stripe-mock seems to take it that `month` is part of the actual endpoint.

Reply:
Can you add a comment to the test to explain that the exception is caused by a stripe-mock limitation and that the test should be updated once stripe-mock's behavior mimics the API's?

Reply:
Added the comment on the different behavior